tests/unit/interfaces/api/test_architecture_boundaries.py

```
@@ -206,7 +206,21 @@ def test_app_factory_registers_legacy_and_api_routes(tmp_path):
     from interfaces.main import create_app
 
     app = create_app(BackendSettings(frontend_dir=tmp_path / "dist"))
-    routes = {route.path for route in app.routes}
+
+    def _collect_paths(routes):
+        """兼容新旧 Starlette：新版会把 include 的路由包成无 .path 的 _IncludedRouter。"""
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)